package.json

```
@@ -1,20 +1,52 @@
 {
   "name": "css-frameworks-ca",
-  "version": "1.0.0",
+  "version": "0.0.3",
```

```suggestion
  "version": "1.0.3",
```

Version numbers never go backwards